values.yaml

```diff
@@ -79,140 +89,146 @@ global:
+  # and connect injector.
   enableConsulNamespaces: false
 
   # Configure ACLs.
   acls:
 
     # If true, the Helm chart will automatically manage ACL tokens and policies
-    # for all Consul and consul-k8s components. This requires Consul >= 1.4 and consul-k8s >= 0.14.0.
+    # for all Consul and consul-k8s components.
+    # This requires Consul >= 1.4 and consul-k8s >= 0.14.0.
     manageSystemACLs: false
 
-    # bootstrapToken references a Kubernetes secret containing the bootstrap token to use
-    # for creating policies and tokens for all Consul and consul-k8s components.
-    # If set, we will skip ACL bootstrapping of the servers and will only initialize
-    # ACLs for the Consul and consul-k8s system components.
-    # Requires consul-k8s >= 0.14.0
+    # A Kubernetes secret containing the bootstrap token to use for
+    # creating policies and tokens for all Consul and consul-k8s components.
+    # If set, we will skip ACL bootstrapping of the servers and will only
+    # initialize ACLs for the Consul clients and consul-k8s system components.
+    # Requires consul-k8s >= 0.14.0.
     bootstrapToken:
+      # The name of the Kubernetes secret.
       secretName: null
+      # The key of the Kubernetes secret.
       secretKey: null
 
     # If true, an ACL token will be created that can be used in secondary
     # datacenters for replication. This should only be set to true in the
     # primary datacenter since the replication token must be created from that
     # datacenter.
     # In secondary datacenters, the secret needs to be imported from the primary
-    # datacenter and referenced via global.acls.replicationToken.
-    # Requires consul-k8s >= 0.13.0
+    # datacenter and referenced via `global.acls.replicationToken`.
+    # Requires consul-k8s >= 0.13.0.
     createReplicationToken: false
 
     # replicationToken references a secret containing the replication ACL token.
     # This token will be used by secondary datacenters to perform ACL replication
     # and create ACL tokens and policies.
-    # This value is ignored if bootstrapToken is also set.
-    # Requires consul-k8s >= 0.13.0
+    # This value is ignored if `bootstrapToken` is also set.
+    # Requires consul-k8s >= 0.13.0.
     replicationToken:
+      # The name of the Kubernetes secret.
       secretName: null
+      # The key of the Kubernetes secret.
       secretKey: null
 
-  # Settings related to federating with another Consul datacenter.
+  # Configure federation.
   federation:
     # If enabled, this datacenter will be federation-capable. Only federation
-    # through mesh gateways is supported.
+    # via mesh gateways is supported.
     # Mesh gateways and servers will be configured to allow federation.
-    # Requires global.tls.enabled, meshGateway.enabled and connectInject.enabled
+    # Requires `global.tls.enabled`, `meshGateway.enabled` and `connectInject.enabled`
     # to be true.
-    # Requires Consul 1.8+.
```

Comment:
Do we want to remove this? If true, it could be helpful to know.

Reply:
good catch, adding back